src/test/fuzz/util/net.cpp

```diff
@@ -36,7 +36,11 @@ CNetAddr ConsumeNetAddr(FuzzedDataProvider& fuzzed_data_provider) noexcept
     } else if (network == Network::NET_IPV6) {
         if (fuzzed_data_provider.remaining_bytes() >= 16) {
             in6_addr v6_addr = {};
-            memcpy(v6_addr.s6_addr, fuzzed_data_provider.ConsumeBytes<uint8_t>(16).data(), 16);
+            auto addr_bytes = fuzzed_data_provider.ConsumeBytes<uint8_t>(16);
+            if (addr_bytes[0] == 0xFC) { // Avoid generating IPv6 addresses that look like CJDNS.
+                addr_bytes[0] = 0x55;
```

Comment:
In d9b95cb1eedb9f165df93dcae0aba96ede40767a, I could check that all CJDNS address starts with [OxFC](https://github.com/cjdelisle/cjdns/blob/master/doc/Whitepaper.md#pulling-it-all-together), so, it switches the first byte to `0x55` to avoid generating it that look like CJDNS. But why specifically `0x55`?

Reply:
There is no reason for 0x55 specifically. I added a comment to clarify:

`// Just an arbitrary number, anything != 0xFC would do.`

Reply:
Got it, thank you!